<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TipDialog</class>
 <widget class="QDialog" name="TipDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tips</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="JaG.qrc">:/small_JaG</pixmap>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="tipTextEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="showTipCheckBox">
       <property name="text">
        <string>Show tips after boot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="prevPushButtom">
       <property name="text">
        <string>Prev</string>
       </property>
       <property name="icon">
        <iconset resource="JaG.qrc">
         <normaloff>:/previous</normaloff>:/previous</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextPushButtom">
       <property name="text">
        <string>Next</string>
       </property>
       <property name="icon">
        <iconset resource="JaG.qrc">
         <normaloff>:/next</normaloff>:/next</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closePushButtom">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="icon">
        <iconset resource="JaG.qrc">
         <normaloff>:/cancel</normaloff>:/cancel</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="JaG.qrc"/>
 </resources>
 <connections/>
</ui>
